app/scripts/lib/state-utils.ts

```diff
@@ -100,6 +100,7 @@ function sanitizeSeedlessOnboardingControllerState(state: FlattenedUIState) {
     'metadataAccessToken',
     'refreshToken',
     'revokeToken',
+    'socialLoginEmail',
```

Comment:
You should replace this with a constant value to make sure that the selector still works. 

https://github.com/MetaMask/metamask-extension/blob/b67d57116cfe51aa7c89981770ef670a94b03241/ui/selectors/seedless-onboarding/social-sync.ts#L28

Reply:
added here: 41d904fe7010defeb89857c3aeb2300d67a727ef